README.md

```diff
@@ -563,12 +655,53 @@ sudo systemctl reload nginx
 
 At this step, you should also setup the SSL for Nginx. [Certbot](https://certbot.eff.org/lets-encrypt/ubuntuxenial-nginx) can be a good option if you want a free SSL certificate.
 
+
+### Apache
```

Comment:
I think it's better to move this section to a dedicated file in `docs/`. Maybe we can add also how to get a valid certificate here as well as modern browsers will make it hard for users to authorize `/etc/ssl/certs/ssl-cert-snakeoil.pem` certificate.

Reply:
should i move all the webserver setup to the extras page or just the ssl setup ?

Reply:
instead of creating cusom guides to generate ssl certificates with certbot  why not just use digtaloceans's guides?

nginx: https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04

apache2:https://www.digitalocean.com/community/tutorials/how-to-secure-apache-with-let-s-encrypt-on-ubuntu-18-04

Reply:
I think the current guide is useful. As a new comer, it's not helpful when you "point to a link", rather continue installation steps helps with the installation.

Reply:
the thing is the instructions are largely the same, its just adding the certificate to postifx.

Reply:
> the thing is the instructions are largely the same, its just adding the certificate to postifx.

For someone with experience, yes. For someone who's new would have difficulties separating the "related" code lines to execute. Just curious, why is it a big deal? I mean, we can keep the current guide as it is...

Reply:
The documention in general needs an upgarde, the less stuff to mainatain outside the focus of the project the better.